src/server/internalServer.h

```diff
@@ -188,9 +188,6 @@ class InternalServer {
 
     class CustomizedResources;
     std::unique_ptr<CustomizedResources> m_customizedResources;
-
-    friend std::unique_ptr<ContentResponse> ContentResponse::build(const InternalServer& server, const std::string& content, const std::string& mimetype);
-    friend std::unique_ptr<Response> ItemResponse::build(const InternalServer& server, const RequestContext& request, const zim::Item& item);
```

Comment:
Should probably be moved in previous commit.

Reply:
`ContentResponse::build()` had to stay friend of `InternalServer` thus far because it accessed `m_root`. Friend declaration of `ItemResponse::build()` could be deleted even earlier.

Reply:
> Friend declaration of `ItemResponse::build()` could be deleted even earlier.

Done